MEAnalysis/python/samples_base.py

```diff
@@ -9,7 +9,7 @@
 xsec[("ttjets", "8TeV")] = 252.89
 xsec[("ttjets", "13TeV")] = 831.76
 
-br_h_to_bb = 0.569
+br_h_to_bb = 0.577
```

Comment:
thanks for updating. do you happen to have a link to a reference to put here? I know I didn't have before but i'm making an effort now to put references where possible.